src/types/index.d.ts

```diff
@@ -218,3 +218,8 @@ export interface VariationsMap {
   },
   dtype: 'array' | 'object'
 }
+
+export interface ItemsTracked {
```

Comment:
Please double check but I believe this also accepts `itemId` optionally. Can we add it here?